.github/workflows/test-build.yml

```diff
@@ -1,18 +0,0 @@
-name: Build site
-
-on:
-  pull_request:
-    branches: [main]
```

Comment:
it says this action runs on a pull requests?

Reply:
https://github.com/ExpediaGroup/expediagroup.github.io/blob/main/.github/workflows/release-gh-pages.yml is run daily but by doing so it picks up the latest changes from medium/github/etc (i.e. only pushes update if there was a change)

Reply:
oops. i was looking at the comments for one workflow and deleted the other :s